voyager_archive/views_utils.py

```diff
@@ -8,22 +8,55 @@ def get_auth_record(auth_id: int) -> list:
 
     return get_marc_fields(marc_text)
 
+
 def get_bib_record(bib_id: int) -> list:
     marc_text = get_object_or_404(BibRecord, bib_id=bib_id).bib_record
 
     return get_marc_fields(marc_text)
 
+
 def get_mfhd_record(mfhd_id: int) -> list:
     marc_text = get_object_or_404(MfhdRecord, mfhd_id=mfhd_id).mfhd_record
-   
+
     return get_marc_fields(marc_text)
 
+
 def get_marc_fields(marc_text: str) -> list:
-    byte_obj = marc_text.encode('utf-8')
+    byte_obj = marc_text.encode("utf-8")
     marc_record = Record(data=byte_obj)
     marc_fields = marc_record.get_fields()
 
-    return marc_fields
+    marc_field_list = []
+    marc_record_dict = {}
+    marc_record_dict["leader"] = marc_record.leader
+
+    for field in marc_fields:
+
+        tmp_rec_dict = {}
+        tmp_rec_dict["tag"] = field.tag
+        if hasattr(field, "data"):
+            tmp_rec_dict["data"] = field.data
+
+        if hasattr(field, "indicator1"):
+            tmp_rec_dict["indicators"] = f"{field.indicator1} {field.indicator2}"
+
+        try:
+            subfield_dict = {}
+            subfield_default_dict = field.subfields_as_dict()
```

Comment:
`subfields_as_dict()` combines multiple values into one list - which is OK at this point, but needs to be handled correctly for display, as noted in my comment above on bib 9411434 and its 040 field.

Confirming for myself :)
```
>>> f040.subfields_as_dict()
defaultdict(<class 'list'>, {'a': ['CLU'], 'b': ['eng'], 'e': ['rda'], 'c': ['CLU'], 'd': ['CLU', 'OCLCO', 'OCLCF']})
```